### DISCUSSION TITLE
[Pop-ups] Section Support

### DISCUSSION THREAD

**Discussion by willliamchan** [Clooos/Bubble-Card#524] [Feature requests]
Knowing the bubble card itself support section and it's been great, is there a way to make pop-up card support section too?

**Comment by Clooos** [comment]
I would love that too, but this is a missing feature from the vertical stack, maybe they are going to add this feature one day.

**Comment by willliamchan** [comment]
No worries, just want to be sure coz I saw from your post that you had yours migrated to Section.

Enjoy the weekend!


________________________________

William Chan



T +44 (0) 789 888 8686

E ***@***.***

________________________________
